lib/components/webview/webview_page.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:harpy/components/components.dart';
+
+class WebviewPage extends ConsumerWidget {
+  const WebviewPage({
+    required this.initialUrl,
+  });
+
+  static const String name = 'webview';
+  static const String path = '/webview';
+
+  final String initialUrl;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final state = ref.watch(webViewProvider(initialUrl));
+    final notifier = ref.watch(webViewProvider(initialUrl).notifier);
+
+    return HarpyScaffold(
+      safeArea: true,
+      child: CustomScrollView(
+        slivers: [
+          HarpySliverAppBar(
+            title: state.hasTitle ? Text(state.title!) : null,
```

Comment:
```suggestion
            title: state.hasTitle
                ? Text(state.title!, overflow: TextOverflow.ellipsis)
                : null,
            fittedTitle: false,
```
Since the title can be indefinitely long, it's a better idea to not have it in a FittedBox in the app bar and instead have the text overflow